### PR TITLE
update staking launchpad to use Holesky testnet [Fixes #13168]

### LINKED ISSUE #13168: Solo staking page: Goerli has been deprecated; use Holesky
### Describe the bug

The [`/staking/solo`](https://ethereum.org/en/staking/solo/#get-started-on-the-staking-launchpad) page has a widget for users to get started with staking, but it is still pointing to the Goerli Launchpad. This should be updated to use the [Holesky testnet Launchpad](https://holesky.launchpad.ethereum.org/en/) instead.

### To reproduce

1. Go to https://ethereum.org/en/staking/solo/#get-started-on-the-staking-launchpad
2. Click on the dropdown for "Choose a network"
3. See use of Goerli testnet, which has been deprecated.

### Expected behavior

References to Goerli should be removed and replaced with the Holesky testnet where users can practice staking before going to Mainnet.

### Screenshots

_No response_

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by tutizaraz**
## Description

Updated the solo staking page to reflect that Goerli has been deprecated and replaced with Holesky. 

Im more than confident that the translations need to be updated. I would appreciate it if you could suggest the correct approach. Considering the documentation, I didn’t quite understand how to do it correctly.

## Related Issue

[Fixes #13168]

### CODE REVIEW

File: src/intl/en/page-staking.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
   "page-staking-how-solo-works-item-4": "Generate your keys and load them into your validator client",
   "page-staking-how-solo-works-item-5": "Monitor and maintain your node",
   "page-staking-launchpad-widget-testnet-label": "Goerli testnet",
```
Comment (wackerow) [inline src/intl/en/page-staking.json:150]: @tutizaraz We'd want to update this one to "Holesky" as well

**Review by wackerow — CHANGES_REQUESTED** [review]
You're correct that the non-English versions will also need this updated. Typically we handle that through [Crowdin](https://crowdin.com/project/ethereum-org/) where we push English updates, get them translated, and then pull them back in. 

In this case, I think we can probably do both at the same time and just make the updates here.

I would search the repo for all the `page-staking-launchpad-widget-testnet-` strings inside `src/intl/**/page-staking.json` files... The results should look something like this:

```log
src/intl/ar/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "شبكة التجريب Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "بدء المراهنة على شبكة التجريب Goerli",

src/intl/bn/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli টেস্টনেট",
  152:   "page-staking-launchpad-widget-testnet-start": "Goerli টেস্টনেট এ স্টেক করা শুরু করুন",

src/intl/de/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli Testnetz",
  152:   "page-staking-launchpad-widget-testnet-start": "Beginne auf dem Goerli Testnetz zu staken",

src/intl/el/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Δίκτυο δοκιμών Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Ξεκινήστε την αποθήκευση κεφαλαίου στο δίκτυο δοκιμών του Goerli",

src/intl/en/page-staking.json:
  150    "page-staking-launchpad-widget-testnet-label": "Goerli testnet",
  151:   "page-staking-launchpad-widget-testnet-start": "Start staking on Goerli testnet",

src/intl/es/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Red de prueba Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Comience a participar en la red de prueba Goerli",

src/intl/fa/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "شبکه تست Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "سهام‌گذاری در شبکه تست Goerli را شروع کنید",

src/intl/fil/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli testnet",
  152:   "page-staking-launchpad-widget-testnet-start": "Magsimulang mag-stake sa Goerli testnet",

src/intl/fr/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Réseau de test Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Commencer à mettre en jeu sur le réseau de test Goerli",

src/intl/hi/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli टेस्टनेट",
  152:   "page-staking-launchpad-widget-testnet-start": "Goerli टेस्टनेट पर स्टेक करना शुरू करें",

src/intl/hu/page-staking.json:
  149    "page-staking-launchpad-widget-testnet-label": "Goerli-teszthálózat",
  150:   "page-staking-launchpad-widget-testnet-start": "Letétbe helyezés a Goerli-teszthálózaton",

src/intl/id/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Jaringan percobaan Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Mulai penaruhan di jaringan percobaan Goerli",

src/intl/it/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Rete di prova di Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Inizia a mettere in staking sulla rete di prova di Goerli",

src/intl/ja/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli testnet",
  152:   "page-staking-launchpad-widget-testnet-start": "Goerli testnet でステーキングを開始する",

src/intl/ko/page-staking.json:
  119    "page-staking-launchpad-widget-testnet-label": "Goerli 테스트넷",
  120:   "page-staking-launchpad-widget-testnet-start": "Goerli 테스트넷에서 스테이킹 시작",

src/intl/nl/page-staking.json:
  119    "page-staking-launchpad-widget-testnet-label": "Goerli-testnet",
  120:   "page-staking-launchpad-widget-testnet-start": "Start met staken op het Goerli-testnet",

src/intl/pl/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Sieć testowa Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Zacznij stakować na sieci testowej Goerli",

src/intl/pt-br/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Rede de testes Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Comece a participar na rede de testes Goerli",

src/intl/ru/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Тестовая сеть Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Начать стейкинг в тестовой сети Goerli",

src/intl/tr/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli test ağı",
  152:   "page-staking-launchpad-widget-testnet-start": "Goerli test ağında hisselemeye başlayın",

src/intl/uk/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Тестова мережа Goerli",
  152:   "page-staking-launchpad-widget-testnet-start": "Почніть робити ставки в тестовій мережі Goerli.",

src/intl/vi/page-staking.json:
  146    "page-staking-launchpad-widget-testnet-label": "Mạng thử nghiệm Goerli",
  147:   "page-staking-launchpad-widget-testnet-start": "Bắt đầu đặt cọc trên mạng thử nghiệm Goerli",

src/intl/zh/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli 测试网",
  152:   "page-staking-launchpad-widget-testnet-start": "开始在 Goerli 测试网上质押",

src/intl/zh-tw/page-staking.json:
  151    "page-staking-launchpad-widget-testnet-label": "Goerli 測試網",
  152:   "page-staking-launchpad-widget-testnet-start": "開始在 Goerli 測試網上進行質押",
```

Just replace `Goerli` in each case here with `Holesky`. I can help make the necessary changes on the Crowdin side.

File: src/intl/en/page-staking.json (hunk 1 above)
Comment (wackerow) [reply src/intl/en/page-staking.json:150]:
```suggestion
  "page-staking-launchpad-widget-testnet-label": "Holesky testnet",
```

**Comment by SnehalSrivastava27** [comment]
https://github.com/ethereum/ethereum-org-website/pull/13187
@wackerow made all the required changes in this pull request

**Comment by tutizaraz** [comment]
@wackerow as I understand, there is no point in continuing with this pull request, right?

**Comment by tutizaraz** [comment]
Thanks for your assistance! I've made some additional revisions to the translation text. It seems now the decision is yours regarding which pull request to accept and which to decline. I’m here to help if you need it!

**Comment by wackerow** [comment]
Closing out in lieu of #13187, thanks again @tutizaraz!

**Pull request closed without merging**